index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&family=Nunito&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="icon" href="images/GDC.png" />
+    <title>Game Developers Conference (GDC) 2022</title>
+  </head>
+  <body>
+    <nav class="black-nav">
+      <div class="black-nav-content">
+        <img
+          class="blck-nav-img"
+          src="images/facebook.png"
+          alt="facebook-icon"
+        />
+        <img class="blck-nav-img" src="images/twitter.png" alt="twitter-icon" />
+        <p class="blck-nav-text">English</p>
+        <p class="blck-nav-text">My Page</p>
+        <p class="blck-nav-text">Logout</p>
+      </div>
+    </nav>
+    <nav class="navbar">
+      <img class="Logo" src="images/GDC.png" alt="Logo" />
+      <div class="navbar-links">
+        <ul>
+          <li><a href="about.html">ِِAbout</a></li>
```

Comment:
- [ ]  As required by the first reviewer. The link to the About page is currently still not working, both locally and on GitHub pages.  Kindly fix this issue and make sure it works :factory_worker: